### DISCUSSION TITLE
Watchlist Integration on Dev

### DISCUSSION THREAD

**Discussion by ccigas** [sct/overseerr#2990] [Help]
I just upgraded to the dev branch to start testing the Watchlist integration. I know it's dev and there's going to be issues, but I'd like to know what we should expect from it. I wasn't able to find any comments or anything explaining the new feature so forgive me if I missed it. 

Currently on my homepage I am not seeing my watchlist load. Is there any other prep work I need to do to get it loaded up?  

I went through my settings also and allowed auto requests for the Plex Watchlist. I already had auto approve on so I assume once my watchlist loads, I should start seeing requests?  

Will local Plex users get their watchlist auto requested through my Plex account?  

This next question isn't really for Watchlist but I thought I'd ask anyway. I only just noticed the auto approve is for non 4k content. I only have one server for radarr and sonarr that does both 4k and non 4k content. The alert when enabling 4k states that I shouldn't do that so I unselected radarr and sonarr as 4k. So my question here is how do I allow auto requests and auto approvals for 4k content? And why is it currently not allowed for a set up like mine? 

Thank you.

Edit: looks like restarting fixed my watchlist not populating but at least in the first few minutes of my watchlist populating on overseerr, the missing media was not requested.

**Comment by TheCatLady** [comment]
- There are new user permissions and new user settings associated with the new Plex Watchlist integration. Users must have the Auto-Request permissions and enable the new Auto-Request settings in order for media to be automatically requested. Media is requested when the Plex Watchlist Sync job runs every 10 minutes.
- There are separate Auto-Approve permissions for 4K. 4K permissions do not apply if you only have a single Sonarr/Radarr server. Please see https://docs.overseerr.dev/using-overseerr/settings#services
- Managed Plex users will not have access to the Plex Watchlist feature.

**Reply by JourneyOver** [reply]
So I set this all up and only the first 20 items from my plex watchlist are actually imported into the `Your Plex Watchlist`.. If I add another item to my plex watchlist it adds it to overseerr but it removes the oldest item in the `Your Plex Watchlist` as well so it always stays at 20 items..

Am I to assume this is the intentional design choice, and that it just will not load the entire plex watchlist that I have?
